libraries/Microsoft.Bot.Builder.AI.LUIS/LuisExtensions.cs

```diff
@@ -29,7 +29,7 @@ public static IConfigurationBuilder UseLuisSettings(this IConfigurationBuilder b
             var luisRegion = configuration.GetValue<string>("LUIS_AUTHORING_REGION") ?? configuration.GetValue<string>("region") ?? "westus";
             var environment = configuration.GetValue<string>("environment") ?? Environment.UserName;
             var settings = new Dictionary<string, string>();
-            settings["luis:endpoint"] = $"https://{luisRegion}.api.cognitive.microsoft.com";
+            settings["luis:endpoint"] = configuration.GetValue<string>("luis:endpoint") ?? $"https://{luisRegion}.api.cognitive.microsoft.com";
```

Comment:
>  settings["luis:endpoint"] = configuration.GetValue<string>("luis:endpoint") ? [](start = 10, length = 79)

No, I think we shouldn't be passing in luis:endpoint, we should be using authoring regions to compute the endpoint
 switch(authoringRegion)
 {
  case "chinaeast2":
   endpoint = $"https://{authoringRegion}.api.cognitive.azure.cn";
   break;
  default:
   endpoint =  $"https://{authoringRegion}.api.cognitive.microsoft.com";
   break;
 }

Reply:
@tomlm based on the conversation in Team, I believe I don't need to resolve this now...